bloom-housing/bloom
# fix: update dependencies for backend

# Pull Request Template

Updates some out of date dependencies that may or may not have security flags against them

## Description

These are all dependencies that synk has attempted to upgrade for Doorway. This PR just manually does it in core so that it is kept in sync with Doorway

## How Can This Be Tested/Reviewed?

**PG and axios**: Minor versions - Tested by making sure the app runs and API calls are made
google translate: Major version - test that changing to a different language on the listings page still works
swagger-ui: minor version - `/docs` page should still work
new-relic: we don't actively use this but don't want to delete the code in the codebase

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
